src/main/java/homework/task1/ConsolePrinter.java

```diff
@@ -0,0 +1,21 @@
+package homework.task1;
+import homework.ConsoleColors;
+
+public class ConsolePrinter {
+    public static void main(String[] args){
+
+        if(args.length > 0) {
```

Comment:
Opt: if args < 1, for cycle will not have any iteration. So you don't need this if condition